package.json

```diff
@@ -29,28 +28,34 @@
     "govuk_template_jinja": "^0.22.3",
     "helmet": "^3.8.1",
     "jquery": "^3.2.1",
-    "mocha": "^3.5.3",
-    "mochawesome": "^2.3.1",
     "nunjucks": "^3.0.1",
     "require-directory": "^2.1.1",
     "serve-favicon": "^2.4.4",
     "ts-node": "^3.3.0",
     "tsconfig-paths": "^2.3.0",
-    "typescript": "^2.5.2"
+    "typescript": "^2.5.2",
+    "xss-filters": "^1.2.7"
```

Comment:
just adding library to deps won't make people use it imho. I think it would be worth to either use it in sample code or update README

Reply:
As discussed, I removed the library - it was last updated too long ago and might be dead